### PR TITLE
implementing alwaysShowPlaceholder prop

### PR DESCRIPTION
**Pull request by mrmello**
As mentioned in this [issue](https://github.com/TeamWertarbyte/material-ui-chip-input/issues/252)

screenshot from 2018-12-06 13-54-22

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ import ChipInput from 'material-ui-chip-input'
 |onDelete|`func`||Callback function that is called when a new chip was removed (in controlled mode).|
 |onUpdateInput|`func`||Callback function that is called when the input changes.|
 |placeholder|`string`||A placeholder that is displayed if the input has no values.|
+|alwaysShowPlaceholder|`bool`||If true, the placeholder will always appear after inserting new chips.|
```
Comment (leMaik) [inline README.md]: This should be consistent with the other description, so "If true, the placeholder will always be visible.".

Comment (mrmello) [reply README.md]: Done. But the tests are failing and it is not related to my changes =/

**Comment by leMaik** [comment]
@mrmello Thank you for this nice first contribution. :tada:

**Comment by leMaik** [comment]
I don't know why this was closed. :man_shrugging:

**Comment by leMaik** [comment]
@mrmello Wow, I screwed this one up pretty badly. I'm sorry, but at least your name is in the commit history. :)

**Comment by mrmello** [comment]
@leMaik haha
Thanks! no problem :)

**Pull request closed without merging**